backend/integrations/discord/views.py

```
@@ -115,6 +122,7 @@ async def skip(self, interaction: discord.Interaction, button: discord.ui.Button
             item.disabled = True
         try:
             await interaction.response.edit_message(view=self)
+        except discord.NotFound:
+            logger.debug(f"Message not found when editing skip view for user {interaction.user.id}")
         except Exception:
-            # If edit fails (e.g., message deleted), ignore
-            pass
+            logger.exception(f"Unexpected error editing skip view")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)